windows/RNCamera/RCTCameraModule.cs

```diff
@@ -90,35 +87,58 @@ public override string Name
         public CameraForViewManager CameraManager { get; } = new CameraForViewManager();
 
         [ReactMethod]
-        public async void capture(JObject options, IPromise promise)
+        public async void checkMediaCapturePermission(IPromise promise)
         {
-            var viewTag = options.Value<int>("view");
+            MediaCapture newCapture = new MediaCapture();
+            try
+            {
+                await newCapture.InitializeAsync();
+            }
+            catch
+            {
+                promise.Resolve(false);
+                return;
```

Comment:
debug print?

Reply:
Same, a yellow box will be shown in debugger mode for RN app with the error message.